### PR TITLE
Add arbitrum broadcast file

### PR DESCRIPTION
**Pull request by MartinquaXD**
# Description
Adds broadcast file for deployment on arbitrum.
All contracts deployed and verified.

### CODE REVIEW

**Comment by MartinquaXD** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MartinquaXD** [comment]
recheck

**Review by fleupold — DISMISSED** [review]
A bit mundane, but could you also update the deployment table in the README.md (we may revamp it to simply list the contract address once and then document which networks it is deployed on)

File: README.md [hunk 1]
```diff
@@ -123,16 +123,16 @@ Fortunately, when using Safe, it is possible to batch together all the above cal
...
+| `ExtensibleFallbackHandler`    | [0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5](https://etherscan.io/address/0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5) | [0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5](https://goerli.etherscan.io/address/0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5) | [0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5](https://gnosisscan.io/address/0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5) | [0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5](https://sepolia.etherscan.io/address/0x000000000000000000000000FFFFFFFFFFFFFFFF) | [0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5](https://arbiscan.io/address/0x2f55e8b20D0B9FEFA187AA7d00B6Cbe563605bF5) |
```
[the remark this answers is not in the document]
Comment (MartinquaXD) [reply README.md]: Good catch. Not sure what happened there.

**Review by fleupold — APPROVED** [review]

**Review by mfw78 — APPROVED** [review]

**Pull request merged**